llvm/include/llvm/CodeGen/DFAPacketizer.h

```diff
@@ -70,7 +74,7 @@ class DefaultVLIWScheduler : public ScheduleDAGInstrs {
   void postProcessDAG();
 };
 
-class DFAPacketizer {
+class DFAPacketizer : public ResourceCycle {
```

Comment:
UPSTREAM NOW

The refactoring makes sense.

Reply:
I have an uncommited PR related to this that virtualizes ResourceManager in the Pipeliner rather than have a UseDFA flag.